internal/commands/push.go

```diff
@@ -44,7 +45,8 @@ func pushCmd(dockerCli command.Cli) *cobra.Command {
 	}
 	flags := cmd.Flags()
 	flags.StringVarP(&opts.tag, "tag", "t", "", "Target registry reference (default: <name>:<version> from metadata)")
-	flags.StringSliceVar(&opts.platforms, "platform", nil, "For multi-arch service images, only push the specified platforms")
+	flags.StringSliceVar(&opts.platforms, "platform", []string{"linux/amd64"}, "For multi-arch service images, only push the specified platforms")
```

Comment:
Rephrase the help message? As it not "only push" anymore, but just "push the specified platforms" ?

Reply:
Fixed